<launch>

  <rosparam file="$(find mpu_6050_reader)/config/test_config.yaml" />

  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" > 
     <arg name="port" value="8080"/>
	<!-- <arg name="address" default="albepi.local" /> -->
  </include>

  <node name="reader" pkg="mpu_6050_reader" type="reader.py" cwd = "node"  output="screen" />
  <node pkg="rosbag" type="record" name="recorder" args="record -a -O $(find mpu_6050_reader)/log/log.bag"/>

</launch>